seqr/views/apis/anvil_workspace_api.py

```diff
@@ -155,11 +154,19 @@ def create_project_from_workspace(request, namespace, name):
         project, individual_records=pedigree_records, user=request.user
     )
 
-    # Send an email to all seqr data managers
+    # Upload sample IDs to a file on Google Storage
+    ids_path = _get_loading_project_path(project, request_json['sampleType']) + '/{guid}_ids.txt'.format(guid=project.guid)
     try:
-        _send_load_data_email(project, updated_individuals, data_path, request_json['sampleType'], request.user)
+        save_data_to_gs(ids_path, '\n'.join(['s'] + [individual.individual_id for individual in updated_individuals]),
+                        user=request.user)
     except Exception as ee:
-        message = 'AnVIL loading request email exception: {}'.format(str(ee))
+        logger.error('Uploading sample IDs to Google Storage failed. Errors: {}'.format(str(ee)), request.user)
```

Comment:
we also need to log the sample IDs somewhere so we can manually create the file. We will be reporting success to the user here and they will expect us to be able to load their data. You can use the `detail` kwargs for `logger.error` to pass it to the log json

Reply:
It makes sense.